### PR TITLE
Prov 76, add the ability to edit supervisor relationships

### PR DESCRIPTION
**Pull request by cioan**
Hi @mogoodrich , can you please review these changes? Thanks!

### CODE REVIEW

File: omod/src/main/webapp/pages/providermanagement/editProvider.gsp [hunk 1]
```diff
@@ -107,6 +142,43 @@
 }
 </style>
 
+<div id="add-supervisor-dialog" class="dialog" style="display: none">
+    <div class="dialog-header">
+        <h3>${ ui.message("Create Relationship") }</h3>
```
Comment (mogoodrich) [inline omod/src/main/webapp/pages/providermanagement/editProvider.gsp]: We should change all of these into message codes so we can localize...

**Comment by mogoodrich** [comment]
I forget, did we ever post on talk about this? We should let people know this is here when we are ready.

**Comment by mogoodrich** [comment]
Also would be good (in a comment?) to provide a sample json config people could use to set up the app?

File: omod/src/main/webapp/pages/providermanagement/editProvider.gsp (hunk 1 above)
Comment (cioan) [inline omod/src/main/webapp/pages/providermanagement/editProvider.gsp]: Thanks @mogoodrich ! I will add message codes in the next pull request.

**Comment by cioan** [comment]
I have just added a Talk post about this. https://talk.openmrs.org/t/using-codedconceptdatatype/9987

On Feb 7, 2017, at 3:44 PM, Mark Goodrich <notifications@github.com<mailto:notifications@github.com>> wrote:


I forget, did we ever post on talk about this? We should let people know this is here when we are ready.

**Pull request merged**